gnd/src/main/java/com/google/android/gnd/persistence/local/LocalValueStore.java

```diff
@@ -72,4 +73,19 @@ public void saveMapType(int type) {
   public int getSavedMapType(int defaultType) {
     return preferences.getInt(MAP_TYPE, defaultType);
   }
+
+  /**
+   * This method will return whether the user have accepted the terms or not.
```

Comment:
Please remove "This method" here and throughout. i.e. "Returns ..."

"the user have" --> "the currently logged in user"

Reply:
fixed in https://github.com/google/ground-android/pull/845